gui.py

```diff
@@ -231,12 +268,93 @@ def doTimestep(self):
         self.updateLabels()
         self.window.update()
 
+    def doNetworkMenu(self, *args):
+        if self.activeNetwork.get() != "None":
+            self.widgets["agentColorButton"].configure(state="disabled")
+            self.widgets["environmentColorButton"].configure(state="disabled")
+        else:
+            self.widgets["agentColorButton"].configure(state="normal")
+            self.widgets["environmentColorButton"].configure(state="normal")
+        self.destroyCanvas()
+        self.configureCanvas()
+        self.configureEnvironment()
+        self.window.update()
+
     def doWindowClose(self, *args):
         # Indicate to per-timestep rendering to stop rendering
         self.stopSimulation = True
         self.window.destroy()
         self.sugarscape.toggleEnd()
 
+    def drawLines(self):
+        lineCoordinates = set()
+        if self.activeNetwork.get() == "Neighbors (Von Neumann)":
```

Comment:
Can this be extended to cover both neighborhood modes?

If the neighborhood mode is von Neumann, show the von Neumann neighborhood. Otherwise, the Moore neighborhood.

Reply:
Sure, added.